index.html

```diff
@@ -229,16 +229,31 @@ <h3>Project name goes here</h3>
                                    <li class="langs-li-dsk cursor"><p>JavaScript</p></li>
                                </ul>
                            </div>
-                           <div class="btn-group project-view">
-                            <span class="project-view-union cursor open-modal" data-open="modal2">See this project</span>
+                           <div class="project-view">
+                                <span class="project-view-union cursor open-modal" data-open="modal2">See this project</span>
                             
-                            <span><img src="images/Union.png" alt="union"></span>
-                    </div>
+                                <span><i class="fa fa-arrow-right"></i></span>
+                           </div>
                        </div>
                     </div>
                     <div class="grid-item item3">
-                        <div id="img-dsk-3-1">
-                         <img src="images/Rectangle2.png" alt="img">
+                        <div class="img-dsk-3-1">
+                          <div class="project-lg-1-dsk">
+                               <h3>Project name goes ...</h3> 
+                               
+                               <div class="list-languages">
+                                   <ul>
+                                       <li class="langs-li-dsk cursor"><p>HTML/CSS</p></li>
+                                       <li class="langs-li-dsk cursor"><p>Ruby on Rails</p></li>
```

Comment:
- [ ] It would be nice for this text "Ruby on Rails" to be on one line just like the figma design this can be done by increasing the width of the main container of all the content for each project👍🏿 

 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![projects](https://user-images.githubusercontent.com/69638013/157826092-3c357588-5843-4570-a8a0-e9ed83054bb2.png)  | ![Screenshot_1](https://user-images.githubusercontent.com/69638013/157825972-3cb62d6c-7111-4550-bf55-d39f784d07cf.png)